components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -107,29 +104,13 @@ end
 ---@param args table
 ---@return boolean
 function CustomLeague:liquipediaTierHighlighted(args)
-	if (
-		String.isNotEmpty(args.liquipediatiertype) or
-		tonumber(args.liquipediatier) ~= TIER_1
-	) then
-		return false
-	end
-
-	return self:containsPsyonix('organizer') or
-		self:containsPsyonix('sponsor')
-end
-
----@param prefix string
----@return boolean
-function CustomLeague:containsPsyonix(prefix)
-	return Table.any(
-		League:getAllArgsForBase(self.args, prefix),
-		function (_, value) return value == PSYONIX end
-	)
+	return Logic.readBool(self.data.publishertier)
 end
 
 ---@param args table
 function CustomLeague:customParseArguments(args)
 	self.data.rlcsPremier = args.series == SERIES_RLCS and 1 or 0
```

Comment:
Is the corresponding variable still in use?

Reply:
sadly yes